shared/chat/conversation/list-area/normal/index.native.js

```diff
@@ -92,7 +202,12 @@ class ConversationList extends React.PureComponent<Props> {
             // Limit the number of pages rendered ahead of time (which also limits attachment previews loaded)
             windowSize={5}
             removeClippedSubviews={true}
+            forwardedRef={this._listRef}
+            onScrollToIndexFailed={this._onScrollToIndexFailed}
           />
+          {!this.props.containsLatestMessage && this.props.messageOrdinals.size > 0 && (
+            <JumpToRecent onClick={this._jumpToRecent} style={styles.jumpToRecent} />
```

Comment:
It'd be nice if this didn't flash in after searching regardless of whether we have the latest message